sim/paladin/paladin.go

```diff
@@ -167,9 +176,14 @@ func (paladin *Paladin) getPrimarySealSpell(primarySeal proto.PaladinSeal) *core
 
 func (paladin *Paladin) applySeal(newSeal *core.Aura, judgement *core.Spell, sim *core.Simulation) {
 	const lingerDuration = time.Millisecond * 400
+	const lingerDuration6pcT1 = time.Second * 6
```

Comment:
I wonder if you could set `lingerDuration` as an attribute on the paladin struct, then in the Lawbringer Radiance 6pc definition instead just override that attribute with `time.Second * 6`? That would cut down on the work done here and having to check for the set bonus in `paladin.go`